layouts/episode/single.html

```diff
@@ -4,9 +4,9 @@
     <div class="col">
       <h1>{{ title .Title }}</h1>
       {{- if ne $.Params.upcoming true -}}
-        <small>Posted on {{ dateFormat "Monday, Jan 2, 2006" .Date }}</small>
+        <small>Posted on {{ dateFormat "Monday, Jan 2, 2006" .Date }} | {{ partial "categories.html" .}} | {{ partial "series.html" .}} | {{ partial "tags.html" .}}</small>
```

Comment:
I think you're right that an overall partial for categories/series/tags is a smart move. The partial can then check to see if any of those are set and only display if they are (we need a check or else it will break if they aren't set)